packages/flutter/test/widget/flex_test.dart

```diff
@@ -65,7 +65,7 @@ void main() {
           key: flexKey
         )
       ));
-      RenderBox renderBox = tester.findElementByKey(flexKey).renderObject;
+      RenderBox renderBox = tester.elementTreeTester.findElementByKey(flexKey).renderObject;
```

Comment:
We might want to have a `tester.renderObjectOf(...)` function similar to the `tester.stateOf` function.